src/main/java/me/xmrvizzy/skyblocker/mixin/HandledScreenMixin.java

```diff
@@ -80,6 +96,92 @@ protected HandledScreenMixin(Text title) {
         return skyblocker$experimentSolvers$getStack(slot, stack);
     }
 
+    @Inject(method = "drawMouseoverTooltip", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/gui/DrawContext;drawTooltip(Lnet/minecraft/client/font/TextRenderer;Ljava/util/List;Ljava/util/Optional;II)V"), cancellable = true)
+    private void skyblocker$addTooltipComponent(DrawContext context, int x, int y, CallbackInfo ci) {
```

Comment:
Can you merge this with the other mixin `skyblocker$drawMouseOverTooltip`? You can switch the other injection point to this since you have a better injection point here.